File: src/main/java/it/contrader/dto/ClientDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ClientDTO {

	private int idclient;
	
//	private int idmoderator;
	private UserDTO user;

	private String username;
	
	private String password;

	private String scode;
	
	private String email;
	
	
}

File: src/dto/clientdto.ts
import { UserDTO } from './userdto';

/**
 * Classe DTO di User. DEVE essere uguale (stesso nome classe, stessi attributi e stessi nomi) a
 * quello nel backend. 
 * 
 * @author PattyGè
 */
export class ClientDTO {

    id: number;

    userDTO : UserDTO;
	
	username: string;
	
	password: string;

	email: string;

    
}